sdk/communication/communication-email/package.json

```diff
@@ -38,19 +38,20 @@
     "email"
   ],
   "dependencies": {
-    "@azure/communication-common": "^2.3.1",
-    "@azure/core-auth": "^1.9.0",
-    "@azure/core-client": "^1.9.2",
-    "@azure/core-lro": "^2.5.0",
-    "@azure/core-rest-pipeline": "^1.18.0",
-    "@azure/core-util": "^1.11.0",
-    "@azure/logger": "^1.1.4",
+    "@azure/abort-controller": "workspace:^",
+    "@azure/communication-common": "workspace:^",
+    "@azure/core-auth": "workspace:^",
+    "@azure/core-client": "workspace:^",
+    "@azure/core-lro": "^2.7.2",
+    "@azure/core-rest-pipeline": "workspace:^",
+    "@azure/core-util": "workspace:^",
+    "@azure/logger": "workspace:^",
```

Comment:
Don't use `workspace:^` for runtime dependencies yet, otherwise release of this package will be blocked by unpublished versions of dependencies in the workspace.